src/System.CommandLine/Command.cs

```diff
@@ -113,26 +82,32 @@ public void AddOption(Option option)
         public void AddGlobalOption(Option option)
```

Comment:
Is weird that now that you are removing `AddOption` we keep `AddGlobalOption`. If we follow the pattern this should be named `AddGlobal`, no? But that sounds weird.

Not asking for a change, just wanted to point that out.

Reply:
That is on my TODO list, we just need to find a good name for "global" options. In the near future it might look like this:

```cs
Option<bool> help = new ("--help);
help.IsCommon = true;
command.Options.Add(help);
```